### PR TITLE
Added psr/log 3

### PR DESCRIPTION
**Pull request by xtrime-ru**

### CODE REVIEW

**Comment by xtrime-ru** [comment]
Dont bump php version

File: composer.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
+        "php": "^8.1",
         "ext-json": "*",
-        "nesk/rialto": "^1.2.0",
-        "psr/log": "^1.0|^3.0",
+        "nesk/rialto": "dev-zoon",
+        "psr/log": "^3.0",
```
Comment (xtrime-ru) [inline composer.json]: Can we keep support of ^1? This is public library, we need it as flexible as possible

File: composer.json [hunk 2]
```diff
@@ -19,10 +19,10 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
+        "php": "^8.1",
```
Comment (xtrime-ru) [inline composer.json]: better keep it low

File: composer.json (hunk 1 above)
Comment (sergkash7) [reply composer.json:25]: No, required for "nesk/rialto": "dev-zoon"

File: composer.json (hunk 2 above)
Comment (sergkash7) [reply composer.json:22]: No, required for "nesk/rialto": "dev-zoon"

**Pull request merged**